docs/ECLLanguageReference/ECLR_mods/Templ-WEBSERVICE.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="_WEBSERVICE">
+  <title>#WEBSERVICE</title>
+
+  <para><emphasis role="bold">#WEBSERVICE<indexterm>
+      <primary>#WEBSERVICE</primary>
+    </indexterm>( </emphasis><emphasis role="bold">[</emphasis><emphasis
+  role="bold">FIELDS</emphasis>(<emphasis>fieldlist</emphasis>),<emphasis
+  role="bold">][</emphasis><emphasis
+  role="bold">HELP</emphasis>(<emphasis>helptext</emphasis>),<emphasis
+  role="bold">][</emphasis><emphasis
+  role="bold">DESCRIPTION</emphasis>(<emphasis>descriptiontext</emphasis>),<emphasis
+  role="bold">]</emphasis> <emphasis role="bold">);</emphasis></para>
+
+  <para><informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="77.25pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">FIELDS</emphasis></entry>
+
+            <entry>The FIELDS parameter specifies field sequence<indexterm>
```

Comment:
You may want to mention that this is an exclusive list.  If you add a FIELDS attribute then only those fields will be used on the form.  This is useful for weeding out fields you don't want to see.